### ISSUE TITLE
Confusing lesson syllabus.

### ISSUE THREAD

**Issue by valentina-s** [swcarpentry/r-novice-inflammation#284]
Looking at the syllabus, I find the list of topics very confusing: sessions 1-8 seem to be continuous. Topics 9-15 seem to start from the beginning with some extra stuff (like factors). Are those considered 'extra' materials. If so, maybe this should be clearly indicated in the title. Or I think it is even better to have a lesson called Supplementary, which links to these materials, so that instructors pace the lesson accordingly - I think the core of the material is in the first few lessons, so they should not be mislead that they have a lot of extra lessons to cover. Let me know what you think a good approach to fix that is.

**Comment by chendaniely** [comment]
The timings on the side refer to how long it would take to cover the entire set of lessons.
Since the workshops are typically 2 days, we devote 1 days worth to `R`.

The later lessons cover certain topics in a bit more detail, so if we do not get to them it's not too bad, but in an ideal situation it all gets covered.

We used to have a section of supplemental materials (and marked as such) but this way as an instructor, they see all the materials that are being used.
For example, some instructors do not teach from `rstudio` but others do, we did not want to force everyone to use rstudio so we created a lesson towards the end after the core material is covered.

I'm not entirely sure how we can better split up the content without making it difficult to find.

Thoughts and comments are welcome

**Comment by gosianow** [comment]
Hello, I had a similar impression as @valentina-s when I looked at the table of contents for the first time. It was a bit confusing for me to understand why some topics repeat and are shuffled like this. Maybe there is no need to rename or change the order of the topics, but such a comment as you @chendaniely gave above about the later lessons being more detailed would clarify it.

**Comment by diyadas** [comment]
Related to #220.

**Comment by bkmgit** [comment]
May be better to indicate dependencies between sections to allow instructors some choice in what to discuss.

**Comment by HaoZeke** [comment]
My understanding is that the lessons are kept in one location here; and the instructors choose the right set of lessons and link to them from the workshop page. We could perhaps add more information on the for instructors portion, but I'm not sure I see this as an issue.

**Comment by aforestsomewhere** [comment]
I agree with @valentina-s that the main headings do read quite disjointed, but also agree with @HaoZeke that there's merit to not amending the overall structure to allow main/supplemental lessons to continue to be linked to respective workshop pages for now (unless a global restructuring of the lesson is in scope).

A note being added to instructor view could be useful, especially to instructors getting ready to teach this one for the first time, to clarify which are the 'core' and 'supplementary' episodes.
